cartoframes/data/observatory/catalog/repository/dataset_repo.py

```diff
@@ -1,11 +1,12 @@
-from .constants import CATEGORY_FILTER, COUNTRY_FILTER, GEOGRAPHY_FILTER, PROVIDER_FILTER, VARIABLE_FILTER
+from .constants import CATEGORY_FILTER, COUNTRY_FILTER, GEOGRAPHY_FILTER, PROVIDER_FILTER
+from ..subscriptions import get_subscription_ids
 from .entity_repo import EntityRepository
-from ..entity import CatalogList
 
+DATASET_TYPE = 'dataset'
 
 _DATASET_ID_FIELD = 'id'
 _DATASET_SLUG_FIELD = 'slug'
-_ALLOWED_FILTERS = [CATEGORY_FILTER, COUNTRY_FILTER, GEOGRAPHY_FILTER, PROVIDER_FILTER, VARIABLE_FILTER]
+_ALLOWED_FILTERS = [CATEGORY_FILTER, COUNTRY_FILTER, GEOGRAPHY_FILTER, PROVIDER_FILTER]
```

Comment:
Why is `VARIABLE_FILTER` removed here?

Reply:
@juanrmn https://github.com/CartoDB/cartoframes/commit/a9984e3fee49d27332d186a25017bcf137052742#diff-513ba1c785c36e0c8c069d9588ca2093

Reply:
I don't remember the reason. Not sure if we decided that makes no sense... but better to know the @juanrmn opinion

Reply:
It's been a time since that, but I think I removed that because `dataset`'s does not have the `variable` attribute, so if I'm not wrong, it would have failed with the previous code anyway. 

Also, to include this filter in the DO API, it would need a quite heavy join between datasets and variables, I think. But please correct me if I'm wrong.

Reply:
OK then. Let's keep it as it is and change it later if necessary.